### PR TITLE
Hw jv add tests for book and category

### PR DESCRIPTION
**Pull request by AdamMudrak**

### CODE REVIEW

**Comment by AdamMudrak** [comment]
image - after BookContollerTest
image - after CategoryControllerTest
image - after BookServiceTest
image - after CategoryServiceTest

File: src/main/java/com/example/onlinebookshop/constants/dtos/CategoryDtoConstants.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.example.onlinebookshop.constants.dto;
+package com.example.onlinebookshop.constants.dtos;
```
Comment (MentorMateAcademy) [inline src/main/java/com/example/onlinebookshop/constants/dtos/CategoryDtoConstants.java]:
```suggestion
package com.example.onlinebookshop.constants.dto;
```

Comment (MentorMateAcademy) [reply src/main/java/com/example/onlinebookshop/constants/dtos/CategoryDtoConstants.java]: don`t change for DTOs

File: src/test/java/com/example/onlinebookshop/BookCategoryConstants.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
+package com.example.onlinebookshop;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class BookCategoryConstants {
```
Comment (MentorMateAcademy) [inline src/test/java/com/example/onlinebookshop/BookCategoryConstants.java:8]: add utilityClass anno from lombok

**Review by MentorMateAcademy — CHANGES_REQUESTED** [review]

**Review by okuzan — APPROVED** [review]

**Pull request merged**